packages/sit-onyx/src/styles/grid.scss

```diff
@@ -2,100 +2,108 @@
 
 // prettier-ignore
 $gridVariants: (
-//  NAME    MAX-VARIANT  COLUMNS  GUTTER  MARGIN
-    "2xs"   none         4        1rem    1rem,
-    "xs"    none         8        1rem    1rem,
-    "sm"    none         8        1.5rem  2rem,
-    "md"    none         12       1.5rem  4rem,
-    "lg"    "md"         16       2rem    4rem,
-    "xl"    "lg"         16       2rem    4rem,
+//  NAME    COLUMNS  GUTTER  MARGIN
+    "2xs"   4        1rem    1rem,
+    "xs"    8        1rem    1rem,
+    "sm"    8        1.5rem  2rem,
+    "md"    12       1.5rem  4rem,
+    "lg"    16       2rem    4rem,
+    "xl"    16       2rem    4rem,
 );
 
+// offset 1 is used so the styles only apply to "grater than breakpoint"
+// so it does not jump around when the browser windows is resized by a few pixels
+$screenOffset: 1;
+
 @layer onyx.utility {
   /**
- * Grid container base class
- */
+  * Grid container base class
+  */
   .onyx-grid {
     display: grid;
     grid-template-columns: repeat(var(--onyx-grid-columns), minmax(0, 1fr));
     gap: var(--onyx-grid-gutter);
-    // Its called margin in the UX concept. So we keep that name for consistency, even when it is implemented through a padding.
+  }
+
+  /**
+  * Page class to apply padding, max-width and optional centered.
+  * Is usually placed at the root element of every page.
+  */
+  .onyx-page__content {
```

Comment:
As this BEM class is defined outside of the context of the `AppLayout` component, we should consider to rename to fit the grid context, e.g.:
- `onyx-grid-container`
- `onyx-grid-page-content`

Reply:
done